adafruit_pyportal.py

```diff
@@ -512,24 +513,33 @@ def get_local_time(self, location=None):
         # pylint: enable=line-too-long
         self._connect_esp()
         api_url = None
+        try:
+            aio_username = secrets['aio_username']
+            aio_key = secrets['aio_key']
+        except KeyError:
+            raise KeyError("\n\nOur image converter service require a login/password to rate-limit. Please register for a freeadafruit.io account and place the user/key in your secrets file under 'aio_username' and 'aio_key'")# pylint: disable=line-too-long
```

Comment:
shouldn't this refer to "time service" not "image converter service" here.

Reply:
woosp fixed